pkg/agent/controller/bgp/controller.go

```diff
@@ -322,6 +329,24 @@ func (c *Controller) getEffectiveBGPPolicy() *v1alpha1.BGPPolicy {
 	return oldestPolicy
 }
 
+func getConfederationConfig(conf *v1alpha1.Confederation) *confederationConfig {
+	if conf == nil {
+		return nil
+	}
+	peers := sets.New[uint32]()
+	for _, v := range conf.MemberASNs {
+		peers.Insert(uint32(v))
+	}
+	return &confederationConfig{
+		identifier: conf.Identifier,
+		peers:      peers,
+	}
+}
+
+func confederationConfigNotChanged(a, b *confederationConfig) bool {
```

Comment:
maybe rename to `confederationConfigEqual` now that you have inverted the logic?

Reply:
Done